include/SemanticModel.h

```diff
@@ -91,7 +88,7 @@ class SemanticModel {
     void makeAttributes(SmallVector<const AttributeSymbol*>& results, const SyntaxList<AttributeInstanceSyntax>& attributes);
 
     const ModuleSymbol* makeModule(const ModuleDeclarationSyntax* syntax, ArrayRef<const ParameterSymbol*> parameters);
-    void handleInstantiation(const HierarchyInstantiationSyntax* syntax, SmallVector<const Symbol*>& results);
+    void handleInstantiation(const Scope* instantiationScope, const HierarchyInstantiationSyntax* syntax, SmallVector<const Symbol*>& results);
```

Comment:
Could you put the scope parameter at the end like all the other functions? Just a nitpick on consistency.